multiapps-controller-core/src/main/java/org/cloudfoundry/multiapps/controller/core/cf/TaggingRequestFilter.java

```diff
@@ -1,14 +1,14 @@
 package org.cloudfoundry.multiapps.controller.core.cf;
 
-import java.io.IOException;
-
 import org.springframework.http.HttpHeaders;
-import org.springframework.http.HttpRequest;
-import org.springframework.http.client.ClientHttpRequestExecution;
-import org.springframework.http.client.ClientHttpRequestInterceptor;
-import org.springframework.http.client.ClientHttpResponse;
+import org.springframework.web.reactive.function.client.ClientRequest;
+import org.springframework.web.reactive.function.client.ClientResponse;
+import org.springframework.web.reactive.function.client.ExchangeFilterFunction;
+import org.springframework.web.reactive.function.client.ExchangeFunction;
+
+import reactor.core.publisher.Mono;
 
-class TaggingRequestInterceptor implements ClientHttpRequestInterceptor {
+class TaggingRequestFilter implements ExchangeFilterFunction {
```

Comment:
Probably rename it to TaggingRequestFilterFunction to be suitable with the interface name